kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/callbacks/CallbackRegistryPunctuatorConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.hypertrace.core.kafkastreams.framework.callbacks;
+
+public class CallbackRegistryPunctuatorConfig {
+  private final long yieldMs;
+  private final long windowMs;
+
+  public CallbackRegistryPunctuatorConfig(long yieldMs, long windowMs) {
+    this.yieldMs = yieldMs;
+    this.windowMs = windowMs;
+  }
+
+  public long getYieldMs() {
```

Comment:
Is there a compelling reason to work in named primitives (e.g. long ms) rather than Duration/Instant here? If not, prefer the object forms for clarity/flexibility?

Reply:
All the configurations in kafka world are typically in ms, since this is at framework level I thought may be have it consistent. We can switch to using Durations I have no concerns there

Reply:
Up to you. I've been burned too many times by inconsistencies with timestamp units - so I hate primitives with a passion - but consistency is good too.